README.md

```diff
@@ -94,6 +94,34 @@ app.use(i18n());
 
 **Middleware dependency** : Language middleware
 
+### IP Filter
+
+Accept or reject requests based upon their IP address.
+Controlled by 2 new environment variables
+
+FILTER_IP = true or false (turn it on or off)
```

Comment:
can we have all the variables names prefixed with IP_FILTER ?

- IP_FILTER_ENABLED
- IP_FILTER_DENIED_IPS
- IP_FILTER_ALLOWED_IPS

This will make all of them stay together in the conf file and make change easier